pkg/manager/member/tikv_member_manager.go

```diff
@@ -511,6 +511,37 @@ func volumeClaimTemplate(r corev1.ResourceRequirements, metaName string, storage
 	}
 }
 
+func transformTiKVConfigMap(src []byte, tc *v1alpha1.TidbCluster) string {
```

Comment:
change `[]byte` to the string, it's better we keep the same type for the input & output and only do the transformation

Reply:
updated.